src/client/common/tooltips/generateContent.js

```diff
@@ -296,7 +322,7 @@ function noDataWarning(name) {
 function filterChemicalFormulas(list) {
   //Filter out Chemical formulas
   if (list instanceof Array) { return list.filter(name => (!name.trim().match(/^([^J][0-9BCOHNSOPrIFla@+\-\[\]\(\)\\=#$]{6,})$/ig))); }
-  return list;
+  return [list];
```

Comment:
so you are returning an array of arrays here?

Reply:
@d2fong 
No, the previous line will return if list was an array. Line 235 actually constructs an array of 1 element, so the information can be passed into the generation functions which all require an Array as in a input. 

List is probably a poor choice for a variable name. The function signature should also be Anything -> Array. The reason why this function catches this error is because pc2 will only return a string if there is only one name entry.